pull/exec_builder.go

```diff
@@ -243,3 +246,41 @@ func (p *Engine) byItemsToDescAndSortExpression(byItems []*util.ByItems) ([]bool
 	}
 	return desc, sortByExprs
 }
+
```

Comment:
I found these utilities handy to visualize the plans—we can exclude them if they're too distracting.